python/setup.py

```diff
@@ -95,16 +95,22 @@ def inc_minor(version):
 
 
 __version__ = get_build_raf_version()
+__raf_version__ = repr(__version__.split("+")[0])
+str_version = repr(__version__)
+str_git = repr(git_version)
```

Comment:
Better to align our names:
```suggestion
__full_version__ = repr(__version__)
__gitrev__ = repr(git_version)
```